labelbox/schema/embedding.py

```diff
@@ -0,0 +1,70 @@
+from typing import Optional, Callable, Dict, Any, List
+
+from labelbox.adv_client import AdvClient
+from labelbox.pydantic_compat import BaseModel, PrivateAttr
+
+
+class EmbeddingVector(BaseModel):
+    embedding_id: str
+    vector: List[float]
+    clusters: Optional[List[int]]
+
+    def to_gql(self) -> Dict[str, Any]:
+        result = {"embeddingId": self.embedding_id, "vector": self.vector}
+        if self.clusters:
+            result["clusters"] = self.clusters
+        return result
+
+
+class Embedding(BaseModel):
```

Comment:
Can we add a short docstring that describes what an embedding is ?

Reply:
Yeah, I might just do this as a follow-up PR, which I mentioned above re: documentation generation.

Reply:
Documentation can be found here: https://github.com/Labelbox/labelbox-python/pull/1543